packages/nc-gui/components/smartsheet/toolbar/CreateSort.vue

```diff
@@ -104,11 +104,11 @@ const onArrowUp = () => {
         @click="onClick(option)"
       >
         <SmartsheetHeaderIcon :column="option" />
-        <NcTooltip class="truncate">
+        <NcTooltip class="truncate" :showOnTruncateOnly="true">
           <template #title> {{ option.title }}</template>
-          <span>
+          <template #default>
```

Comment:
Why did u change `template #default` from `span`?

Reply:
why do we need span tag as we wrap it under a div element already.

`<div ref="el" v-bind="divStyles" @mousedown="onClick">
      <slot />
    </div>
`

Reply:
@musharaf-nocodb
U won't need to explicitly add default template as it's like that by default. Having `template #default` or not should not have a difference. U can skip this comment as its a nitpick